internal/compiler/builtins.slint

```diff
@@ -149,6 +149,7 @@ component WindowItem {
     in property <brush> color <=> background;
     in property <string> title: "Slint Window";
     in property <bool> no-frame;
+    in property <float> resize-border;
```

Comment:
Since this is a value in pixel, it should be a `length`

Reply:
@ogoffart Sorry for the delay but I've pushed this change.